app/src/main/java/io/apicurio/registry/storage/impl/sql/SqlStatements.java

```diff
@@ -182,7 +176,7 @@ public interface SqlStatements {
     /**
      * A statement to get a single artifact (latest version) meta-data by artifactId.
      */
-    String selectArtifactMetaData();
+    public String selectArtifactMetaData();
```

Comment:
`public` in an interface is superficial.

Reply:
True but I was making it consistent within the file.